src/Typescript-codestyle.xml

```diff
@@ -0,0 +1,50 @@
+<code_scheme name="Default (1)" version="173">
```

Comment:
is this file required?

Reply:
Hi Paulo, I've committed this file by accident.
Though it might not be a bad idea perhaps to include a codestyle file in git, it's not related to this PR anyway, so I deleted it again.